### DISCUSSION TITLE
Logging

### DISCUSSION THREAD

**Discussion by stevedanomodolor** [bitcraze/discussions#33] [Q&A] [answered]
Is there any example on how to log data using the library?
Thank you

**Comment by knmcguire** [comment, marked as answer]
hi!

This repository handles the very low level part of the crazyflie communciation, namely handling the packet sending. So logging is a bit too high level example for this repo.

You can try out crazyflie-link-cpp in the crazyflie python library if you set the USE_CFLINK environment variable: https://github.com/bitcraze/crazyflie-lib-python. There you have some examples for logging under examples/logging.

Or you can use the [crazyswarm ](https://github.com/USC-ACTLab/crazyswarm/blob/master/ros_ws/src/crazyswarm/launch/hover_swarm.launch) project that uses the same repo, but that requires to install ros

**Comment by stevedanomodolor** [comment]
Is the crazyflie-tools better than the crazyswam

**Comment by knmcguire** [comment]
That is a difficult question... that really applies to what you are using it for and what your application is to be honest. And we know too little of what you want to achieve.

The quickest to try it out is to use the crazyflie-lib-python.
